manifest.php

```diff
@@ -39,7 +39,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '30.5.3',
+    'version'     => '30.6.3',
```

Comment:
Caution, the version increase should produce an aligned number: `30.5.3` => `30.6.0`